cli.py

```diff
@@ -1,190 +1,95 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
-#
-# This is the first cli prototype for the cabinet core.
-#
-# References;
-#   argparse : https://docs.python.org/3/library/argparse.html
-#   cement   : http://builtoncement.com/2.10/index.html
-#
-# TODO:
-#   - Refactor the code to move each class, utils to its own module.
-
-import signal
-import argparse
-
-from sys import exit
-from cement.core.foundation import CementApp
-from cement.core.controller import CementBaseController, expose
+import click
+
+from utils import get_content_from_editor
 
 from cabinet_wrapper import CabinetWrapper
 
 
-VERSION = '0.0.1'
-BANNER = """
-Cabinet command line v{0}
-GNU GENERAL PUBLIC LICENSE
-""".format(VERSION)
```

Comment:
The version and licence is being removed. I know we are not versioning the cli until reaching a stable version. But, I don't know if this is a good reason for removing the support for it. What do you have in mind?